README.md

```diff
@@ -148,3 +146,37 @@ Before setting up the Titanium SDK, you must generate the appropriate credential
    ```
 
 Cheers!
+
+## Build yourself
+
+### iOS
+
+If you already have Titanium installed, skip the first 2 steps, if not let's install Titanium locally.
+
+1. `brew install yarn --without-node` to install yarn without relying on a specific Node version
```

Comment:
This works perfectly in conjunction with NVM. Select the node version you want to use with NVM, and run the same yarn version in every node version you use. Instead of installing yarn in every node version you install via NVM.